From 78a9a8664ade8e671a5813c27a392506021290a8 Mon Sep 17 00:00:00 2001
From: ejaaskel <esa.jaaskela@suomi24.fi>
Date: Wed, 19 Nov 2025 08:55:53 +0200
Subject: [PATCH] policy/modules/system/systemd: Allow sys_resource for sysctl

The sysctl requires sys_resource capabilities during the boot time.
Loosen up the SELinux policy to allow this.
---
 policy/modules/system/systemd.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index 5823fd543..cb8d3e411 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1766,7 +1766,7 @@ systemd_log_parse_environment(systemd_sessions_t)
 # sys_admin for sysctls such as kernel.kptr_restrict and kernel.dmesg_restrict
 # sys_ptrace for kernel.yama.ptrace_scope
 # net_admin for network sysctls
-allow systemd_sysctl_t self:capability { net_admin sys_admin sys_ptrace };
+allow systemd_sysctl_t self:capability { net_admin sys_admin sys_ptrace sys_resource };
 
 kernel_read_kernel_sysctls(systemd_sysctl_t)
 kernel_request_load_module(systemd_sysctl_t)
-- 
2.34.1

